### ISSUE TITLE
Saving learned progress

### ISSUE THREAD

**Issue by TitanRover** [Code-Bullet/Google-Chrome-Dino-Game-AI#5]
Is there any way of saving the learned progress to a file and then later being able to start/continue the learning from the file?

**Comment by rohanday3** [comment]
Yeah im also trying to figure that out. I think the ai is called a genetic algorithm but i haven't found much online.

**Comment by rohanday3** [comment]
How do you save the genome of the NEAT AI
